x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/optimizer/LogicalPlanOptimizer.java

```diff
@@ -1746,114 +302,4 @@ public final LogicalPlan apply(LogicalPlan plan, P context) {
 
         protected abstract LogicalPlan rule(SubPlan plan, P context);
     }
-
-    /**
-     * Normalize aggregation functions by:
-     * 1. replaces reference to field attributes with their source
-     * 2. in case of Count, aligns the various forms (Count(1), Count(0), Count(), Count(*)) to Count(*)
-     */
-    // TODO still needed?
-    static class NormalizeAggregate extends Rule<LogicalPlan, LogicalPlan> {
```

Comment:
NB: This rule was deleted, not pulled up to a top level class.  There were no usages of it.